portfolio/src/main/java/com/google/sps/data/ArrayListTimeRangeGroup.java

```diff
@@ -5,7 +5,8 @@
 import java.util.List;
 
 /** Models an implementation of the TimeRangeGroup model using ArrayList. */
-public class ArrayListTimeRangeGroup extends AbstractListTimeRangeGroup implements TimeRangeGroup {
+public class ArrayListTimeRangeGroup extends AbstractListTimeRangeGroup
+    implements TimeRangeGroup, Iterable<TimeRange> {
```

Comment:
I think TimeRangeGroup can extend Iterable<TimeRange>, as it exposes the iterator method.

Reply:
I updated the abstract class to implement Iterable<TimeRange>.

Reply:
I would suggest that you make the interface extend iterable, because often you are working with interface and not the impl class. Here are some examples where the interface extends iterable.

https://docs.oracle.com/javase/8/docs/api/java/util/List.html
https://docs.oracle.com/javase/7/docs/api/java/util/Set.html